solidity/contracts/converter/BancorConverterRegistry.sol

```diff
@@ -147,15 +151,12 @@ contract BancorConverterRegistry is IBancorConverterRegistry, ContractRegistryCl
         uint8 _smartTokenDecimals,
         uint32 _maxConversionFee,
         IERC20Token[] memory _reserveTokens,
-        uint32[] memory _reserveWeights,
-        uint256[] memory _reserveAmounts
+        uint32[] memory _reserveWeights
     )
-    public payable
+    public payable returns (IBancorConverter)
     {
         uint256 length = _reserveTokens.length;
-        require(length == _reserveWeights.length);
-        require(length == _reserveAmounts.length);

```

Comment:
@yudilevi : Why have you omitted `require(length == _reserveAmounts.length);`?

Reply:
What do you mean? I completely removed reserve amounts from the function